health-services/transformer/src/main/java/org/egov/transformer/service/BoundaryService.java

```diff
@@ -38,7 +34,7 @@ public class BoundaryService {
 
     private final TreeGenerator treeGenerator;
 
-    private static final Map<String, List<Boundary>> boundaryListMap = new ConcurrentHashMap<>();
+    private static BoundaryTree cachedBoundaryTree = null;
```

Comment:
@kavi-egov instead of maintaining boundarylist in cache, i am maintaining only single tree as cached item